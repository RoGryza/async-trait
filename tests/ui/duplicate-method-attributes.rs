use async_trait::async_trait;

#[async_trait]
pub trait Trait {
    #[async_trait(?Send)]
    #[async_trait(?Send)]
    async fn method();
}

fn main() {}

