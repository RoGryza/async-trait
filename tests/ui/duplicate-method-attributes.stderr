error: duplicate #[async_trait] in method
 --> tests/ui/duplicate-method-attributes.rs:6:5
  |
6 |     #[async_trait(?Send)]
  |     ^^^^^^^^^^^^^^^^^^^^^
